cranelift/interpreter/Cargo.toml

```diff
@@ -11,11 +11,12 @@ readme = "README.md"
 edition = "2018"
 
 [dependencies]
-cranelift-codegen = { path = "../codegen", version = "0.67.0", default-features = false }
+cranelift-codegen = { path = "../codegen", version = "0.67.0", features = ["all-arch"] }
```

Comment:
Yes, we should remove this once the old x86 backend goes away. I only had to enable this because otherwise somewhere in the CI a non-x86 build is triggered that doesn't contain all of the `Opcode::X86...` variants, causing compilation failures.